x-pack/platform/plugins/private/upgrade_assistant/server/routes/status.test.ts

```diff
@@ -36,28 +39,36 @@ jest.mock('../lib/es_system_indices_migration', () => ({
 const getESSystemIndicesMigrationStatusMock = getESSystemIndicesMigrationStatus as jest.Mock;
 
 const esDeprecationsResponse = {
-  cluster: [
+  totalCriticalDeprecations: 1,
+  migrationsDeprecations: [
     {
-      level: 'critical',
-      message:
-        'Model snapshot [1] for job [deprecation_check_job] has an obsolete minimum version [6.3.0].',
-      details: 'Delete model snapshot [1] or update it to 7.0.0 or greater.',
-      url: 'doc_url',
-      correctiveAction: {
-        type: 'mlSnapshot',
-        snapshotId: '1',
-        jobId: 'deprecation_check_job',
-      },
+      // This is a critical migration deprecation object, but it's not used in the tests
```

Comment:
The actual object doesn't matter in this suite, all the test assert for are values and array length. When we do start asserting the mock entries properties, we can add them back. It's less work than having to update them since we're not testing against them.